x/staking/client/rest/query.go

```diff
@@ -154,7 +154,7 @@ func delegatorTxsHandlerFn(cliCtx context.CLIContext, cdc *codec.Codec) http.Han
 			actions = append(actions, staking.MsgBeginRedelegate{}.Type())
 			actions = append(actions, tags.ActionCompleteRedelegation)
 		default:
-			w.WriteHeader(http.StatusNoContent)
```

Comment:
I guess we don't need to modify this since we'll probably delete this function since it's just a txs query (ie. duplicate)

Reply:
Noted. Reverted